### ISSUE TITLE
Feedback from running the course

### ISSUE THREAD

**Issue by cloudhary** [joelchoo/live-parking-info#18]
- [ ] Explain the flow of the function call
- [ ] repeat things twice when presenting, people don't generally get it
- [ ] milestones should be changes, not things staying the same
- [x] explain ===
- [x] change slide 38 to thae in function inputs so that we can tell if things are different
- [x] slide 43 location input variabale name change
- [ ] grabLocation function is unclear
- [ ] bigger printed out notes so that people don't confuse ) and } and ]?
- [ ] people are forgetting to remove addCarpakToPage so they don't see the change happen

**Comment by cloudhary** [comment]
camelCasing is hard, people are getting confused by it. Slide 31 where they are told to do `document.getElementById`, they'll make it `ID` instead of `Id`

**Comment by cloudhary** [comment]
Syntax is a huge issue, we should call them out explicitly. Curly braces are hard to follow in slide 34. We should also explain that bakeCake is the name of the function?

**Comment by cloudhary** [comment]
slide 35 when we introduce functions we should put the inputs into the correct functions so that it makes sense to people how they're used?

**Comment by arshadali172** [comment]
implementing the api call needs to be broken up into more steps. It's too overwhelming to be writing 2 new functions and calling them sequentially in the same step

**Comment by arshadali172** [comment]
maybe we can use dash casing (e.g. `grab-location`) instead of camel casing?

**Comment by cloudhary** [comment]
All issues have been transferred to https://github.com/datagovsg/live-parking-info/issues/1 and will be addressed there
